### PR TITLE
correct spelling

### PR DESCRIPTION
**Pull request by corbolais**
thanks for the utils! very handy :)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+tags
```
Comment (yloiseau) [inline .gitignore]: This file is specific to some tools that are developer dependent, and thus shouldn't be present here. Please put it in your global `.gitignore`.

**Review by yloiseau — CHANGES_REQUESTED** [review]

**Comment by yloiseau** [comment]
Thank you for the contribution. Just a minor fix (see review).

**Comment by corbolais** [comment]
Can fix.

OTOH, I don't see any disadvantage for developers having this line included. It's a [ctags](http://ctags.sourceforge.net/) file (i.e. used w/ a vim plugin or just plain ctags).

**Comment by yloiseau** [comment]
I know what a ctags file is. My point is that this kind of files, that are specific to the developer's environment (IDE projects, .DS_STORE for Mac users, ...) should be ignored in the global `~/.gitignore` file instead of the local project one. The project `.gitignore` should only contain patterns relevant to the project, i.e. ignoring files produced by the build process and so on. This allows to:
1. keep things DRY by ignoring these files only in one place, and don't repeat the list in every projects you have;
2. don't bloat the project configuration with files or patterns that are not useful to other devs, making it easier to grasp for new comers.

**Comment by corbolais** [comment]
good point, thx.
pushed rm commit.

**Review by yloiseau — APPROVED** [review]

**Pull request merged**